packages/nextcloud/test/dashboard_test.dart

```diff
@@ -21,17 +21,12 @@ void main() {
       test('Get widgets', () async {
         final response = await client.dashboard.dashboardApi.getWidgets();
         expect(response.statusCode, 200);
-        expect(
```

Comment:
again; why was some information removed?

Reply:
Not reliable, if you have a minimal version of the server then apps like recommendations are missing and we can not rely on them here.